pkg/backends/otlp/backend.go

```diff
@@ -252,7 +254,7 @@ func (bd *Backend) SendMetricsAsync(ctx context.Context, mm *gostatsd.MetricMap,
 	err := bd.postMetrics(ctx, group.Values())
 	if err != nil {
 		bd.logger.WithError(err).WithFields(logrus.Fields{
-			"endpoint": bd.endpoint,
```

Comment:
we can keep the field as "endpoint" to keep it consistent

Reply:
Hi @hstan, this has been reverted.